application/src/main/java/common/topology/transactional/initializer/OBInitializer.java

```diff
@@ -227,7 +227,7 @@ private void loadOnlineBiddingEvents(BufferedReader reader, int totalEvents, boo
             String[] split = txn.split(",");
             if (split.length > 3) {
                 int npid = (int) (Long.parseLong(split[1]) / partitionOffset);
-                int keyLength = NUM_ACCESS*Transaction_Length;
+                int keyLength = 1*Transaction_Length;
```

Comment:
Why should we put an additional 1* here?

Reply:
In this application, the NUM_ACCESS for bid transaction is one